app/(dashboard)/appSidebar.tsx

```
@@ -63,40 +64,50 @@ export function AppSidebar() {
     }
   };
 
+  const handleBackToApp = () => {
+    const fallback = `/mine`;
+    router.push(previousAppUrlRef.current || fallback);
+    handleItemClick();
+  };
+
+  const renderHeader = () => {
```

ESLint

[error] 'renderHeader' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)